qa/run-standalone.sh

```diff
@@ -24,6 +24,15 @@ else
     COREPATTERN="core.%e.%p.%t"
 fi
 
+function finish() {
+    if [ -n "$precore" ]; then
+        sudo sysctl -w ${KERNCORE}=${precore}
+    fi
+    exit 0
```

Comment:
no need to `exit 0`. and might want to reuse this function at https://github.com/ceph/ceph/pull/17026/files#diff-5e483a2d055861968b99eacaa3688b6fR108.